x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/kibana_instructions.js

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+import {
+  EuiSpacer,
+  EuiCodeBlock,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiCallOut
+} from '@elastic/eui';
+import { formatDateTimeLocal } from '../../../../common/formatting';
+
+export function getKibanaInstructions(product, {
+  doneWithMigration,
+  kibanaUrl,
+  esMonitoringUrl,
+  checkForMigrationStatus,
+  checkingMigrationStatus,
+  hasCheckedMigrationStatus
+}) {
+  const enableCollectionStep = {
+    title: 'Enable monitoring collection',
+    children: (
+      <Fragment>
+        <p>
+          Set the xpack.monitoring.collection.enabled setting to true on each node in the production cluster.
+          By default, it is disabled (false).
+        </p>
+        <EuiSpacer size="s"/>
+        <EuiCodeBlock
+          isCopyable
+          language="curl"
+        >
+          {`
+PUT _cluster/settings
+{
+"persistent": {
+  "xpack.monitoring.collection.enabled": true
+}
+}
```

Comment:
Spacing and brackets looks off on this one

Reply:
Yup, nice catch!